client/src/app.jsx

```diff
@@ -1,23 +1,107 @@
 import React from 'react';
-import ProductDetails from './productDetails';
-import QuestionsAnswers from './qa';
+import axios from 'axios';
+// import ProductDetails from './productDetails';
```

Comment:
Could you comment these components back in?

Reply:
Sure, i'll comment back in all the our components that have working indexes